app/scripts/modules/ecs/pipeline/stages/scaleDownCluster/ecsScaleDownClusterStage.js

```diff
@@ -0,0 +1,69 @@
+'use strict';
+
+const angular = require('angular');
+
+module.exports = angular.module('spinnaker.ecs.pipeline.stage.scaleDownClusterStage', [
+])
+  .config(function(pipelineConfigProvider) {
+    pipelineConfigProvider.registerStage({
+      provides: 'scaleDownCluster',
+      cloudProvider: 'ecs',
+      templateUrl: require('./scaleDownClusterStage.html'),
+      executionDetailsUrl: require('./scaleDownClusterExecutionDetails.html'),
```

Comment:
`scaleDownClusterExecutionDetails.html` is an empty file? If you want to use the default, just omit this line.